src/multidim/setdiff.jl

```diff
@@ -33,7 +33,8 @@ end
 #
 #     end
 # end
-"""
+
+@doc doc"""
```

Comment:
Also probably no longer needs `@doc`.

Reply:
(After the change of `\` to `\setminus` in my PR from yesterday.)

Reply:
I guess the problem was indeed solved yesterday by merging #157. Let me rebase to current master and repush, to see how travis reacts.